packages/core/strapi/lib/services/entity-service/types/params/index.d.ts

```diff
@@ -4,11 +4,13 @@ import type * as Sort from './sort';
 import type * as Pagination from './pagination';
 import type * as Fields from './fields';
 import type * as Filters from './filters';
+import type * as PublicationState from './publication-state';
 
 export type For<TSchemaUID extends Common.UID.Schema> = {
   sort?: Sort.Any<TSchemaUID>;
   fields?: Fields.Any<TSchemaUID>;
   filters?: Filters.Any<TSchemaUID>;
-} & Pagination.Any;
+} & Pagination.Any &
+  PublicationState.For<TSchemaUID>;
```

Comment:
Just curious! why the entity service params type is called "For" ?

Reply:
Namespace param :) 

`Entityservice.Params.For<uid>`

Reply:
💡  makes sense, tyy